### PR TITLE
Add k8s Apps to sandbox job (SRE-59)

### PR DESCRIPTION
**Pull request by nadeemshahzad**

### CODE REVIEW

File: devops/jobs/CreateSandbox.groovy [hunk 1]
```diff
@@ -262,6 +275,9 @@ class CreateSandbox {
                 booleanParam("run_oauth",true,"")
 
                 stringParam("nginx_users",'[{"name": "{{ COMMON_HTPASSWD_USER }}","password": "{{ COMMON_HTPASSWD_PASS }}","state":"present"}]',"")
+
+                booleanParam("argocd",false,"This will install the ArgoCD in minikube in the sandbox")
```
Comment (adzuci) [inline devops/jobs/CreateSandbox.groovy:279]: If you want to you can remove "the" in front of "ArgoCD"

File: devops/jobs/CreateSandbox.groovy [hunk 2]
```diff
@@ -94,6 +94,18 @@ class CreateSandbox {
                         relativeTargetDirectory('configuration-internal')
                     }
                 }
+                git {
```
Comment (adzuci) [inline devops/jobs/CreateSandbox.groovy:97]: @coryleeio pointed out that `configuration-internal` above is actually `edx-internal` so we shouldn't be checking it out twice.

Cory:
```
I also think that configuration-internal and configuration-secure, might actually be edx-internal and
edx-secure at a different level of abstraction and that adding edx-internal there may not be necessary, 
you have to remember that code is in configuration so it’s open source so relying on edx-internal directly 
isn’t appropriate
```

@nadeemshahzad could your logic point at the `configuration-internal` folder instead?

**Review by adzuci — CHANGES_REQUESTED** [review]
I think some of this may not be needed.

**Comment by nadeemshahzad** [comment]
Closing as we don't want to install ArgoCD inside minikube on Sandox.

File: devops/jobs/CreateSandbox.groovy (hunk 2 above)
Comment (adzuci) [reply devops/jobs/CreateSandbox.groovy:97]: @coryleeio I had misunderstood this and thought that `configuration-internal` was `edx-internal`, but it is in fact https://github.com/edx/sandbox-internal so we either need to:

1: Copy the four argocd django application folders out of edx-internal and into sandbox-internal.
2: Clone edx-internal onto the box.

Comment (adzuci) [reply devops/jobs/CreateSandbox.groovy:97]: @nadeemshahzad would like to try #1 first since there is less of a risk of leaking remote config data.

**Review by adzuci — APPROVED** [review]
:shipit:

File: devops/jobs/CreateSandbox.groovy (hunk 2 above)
Comment (adzuci) [reply devops/jobs/CreateSandbox.groovy:97]: That sounds good to me.

**Pull request merged**